brigade-controller/cmd/brigade-controller/controller/handler.go

```diff
@@ -243,6 +245,12 @@ func workerEnv(project, build *v1.Secret, config *Config) []v1.EnvVar {
 			ValueFrom: secretRef("github.token", project),
 		},
 	}
+
+	if psv.String("brigadejsPath") != "" {
```

Comment:
Would it make sense to sanitize the path and check if it actually is a relative path?

Reply:
added!

Reply:
done!